src/SearchParameters/index.js

```diff
@@ -25,17 +19,38 @@ var filterState = require('./filterState');
 
 var RefinementList = require('./RefinementList');
 
+function objectHasKeys(obj) {
+  return Object.keys(obj).length > 0;
+}
+
+/**
+ * isEqual, but only for arrays of base elements, or base elements
+ * @param {number|Array<number>} a first element
+ * @param {number|Array<number>} b second element
+ */
+function isEqualNumArr(a, b) {
```

Comment:
this one is not exactly reusable (on purpose), because it's only used in findArray which is only used in one spot. I preferred here to just make the existing structure work again with isEqual removed, but the better solution is to get rid of both helpers I think. That can be done at a later point?

Reply:
Yes, it could totally be done at a later point :+1: